### ISSUE TITLE
cant set up v2 beta 29

### ISSUE THREAD

**Issue by djixadin** [harlan-zw/nuxt-seo-kit#106]
### Describe the bug

when running npm run dev i get the following error 

[12:00:02 PM]  ERROR  Cannot start nuxt:  pageLocales[locale].includes is not a function

  at setup (node_modules/nuxt-simple-sitemap/dist/module.mjs:811:37)
  at async normalizedModule (node_modules/@nuxt/kit/dist/index.mjs:2141:17)
  at async installModule (node_modules/@nuxt/kit/dist/index.mjs:2426:95)
  at async setup (node_modules/@nuxtseo/module/dist/module.mjs:44:7)
  at async normalizedModule (node_modules/@nuxt/kit/dist/index.mjs:2141:17)
  at async installModule (node_modules/@nuxt/kit/dist/index.mjs:2426:95)
  at async initNuxt (node_modules/nuxt/dist/index.mjs:3549:7)
  at async NuxtDevServer._load (node_modules/nuxi/dist/chunks/dev2.mjs:223:5)
  at async NuxtDevServer.load (node_modules/nuxi/dist/chunks/dev2.mjs:174:7)
  at async NuxtDevServer.init (node_modules/nuxi/dist/chunks/dev2.mjs:169:5)

my package.json 

```
    "@nuxtseo/module": "^2.0.0-beta.29",
    "nuxt": "^3.7.2",

```

nuxt.config.ts
```
  site: {
    identity: {
      type: 'Person'
    },
    twitter: '@harlan-zw',
    url: 'https://example.com',
    name: 'My Sire',
    description: 'some random desc',
    defaultLocale: 'en'
  },
```

**Comment by harlan-zw** [comment]
I think this is because you have a broken `i18n.pages` configuration, would you mind sharing it?

**Comment by djixadin** [comment]
never made it working via a separate file so i inline the config 

```
const localeMap: { [K in Market]: string } = {
  hr: 'hr',
  ke: 'en',
  sr: 'sr'
}

const locale = localeMap[market]
i18n: {
    types: 'composition',
    customRoutes: 'config',
    locales: [{ code: locale, file: `${market}.json` }],
    defaultLocale: locale,
    strategy: 'prefix_except_default',
    lazy: true,
    langDir: 'translations',
    compilation: {
      strictMessage: false
    },
    pages: {
      '[sport]/news/[name]/[id]/index': {
        sr: '/[sport]/vesti/[name]/[id]',
        hr: '/[sport]/vijesti/[name]/[id]',
        en: '/[sport]/news/[name]/[id]'
      },
      '[sport]/news/[name]/[id]/comments/index': {
        sr: '/[sport]/vesti/[name]/[id]/komentari',
        hr: false,
        en: '/[sport]/news/[name]/[id]/comments'
      },
      '[sport]/news/[name]/[id]/comments/new': {
        sr: '/[sport]/vesti/[name]/[id]/komentari/novi',
        hr: false,
        en: '/[sport]/news/[name]/[id]/comments/new'
      },
      'bet-center': {
        sr: '/bet-centar',
        hr: '/bet-centar',
        en: '/betting-news'
      },
      'all-news': {
        sr: '/sve-vesti',
        hr: '/sve-vijesti',
        en: '/latest-news'
      },
      'tags/[id]': {
        sr: '/tagovi/[id]',
        hr: '/tagovi/[id]',
        en: '/tags/[id]'
      },
      'writer/[id]': {
        sr: '/novinar/[id]',
        hr: '/novinar/[id]',
        en: '/writer/[id]'
      },
      'terms-of-use': {
        sr: '/uslovi-koriscenja',
        hr: '/uvjeti-koristenja',
        en: '/terms-of-use'
      },
      'about-us': {
        sr: '/o-nama',
        hr: '/o-nama',
        en: '/about-us'
      },
      'privacy-policy': {
        sr: '/politika-privatnosti',
        hr: '/politika-privatnosti',
        en: '/privacy-policy'
      },
      'for-you': {
        sr: '/za-tebe',
        hr: '/za-tebe',
        en: '/for-you'
      },
      notifications: {
        sr: '/notifikacije',
        hr: '/notifikacije',
        en: '/notifications'
      },
      'my-profile': {
        sr: '/moj-profil',
        hr: '/moj-racun',
        en: '/my-profile'
      }
    }
  },
```

**Comment by harlan-zw** [comment]
I think that `false` is killing it, I'll do some tinkering soon 

```ts
   '[sport]/news/[name]/[id]/comments/new': {
        sr: '/[sport]/vesti/[name]/[id]/komentari/novi',
        hr: false,
        en: '/[sport]/news/[name]/[id]/comments/new'
      },
```

**Comment by harlan-zw** [comment]
Should be fixed in `2.0.0-beta.30`
